### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by BlamoThePollo** [vivinano/MudaeAutoBot#109]
So I think it would be a cool implementation if there was a way to check which user is running the mudae command and if it is a certain user, your bot doesn't attempt to claim from them.. maybe this is too specific but I think it would be useful.
If you don't want to implement this, would you mind giving me the syntax for this check (if possible) ?

**Comment by vivinano** [comment]
Possible yes but not as easy as you might think so you have to tie the message id of 2 separate entities which I’ve saved in message buff 

Basically you have to tie let’s say user id or X person and match it with the message id of the mudae message so basically making a “dictionary” of
Message id : user id and this will be technically be assumption based where the exact next or previous message in the buffer is a mudae message 


TLDR it’s complicated enough that an average user would not need it and would also add more processing as now you have to 
Match user id of $wg to message id from mudae with embeds(complete separate) > make a discord search message call > check user id of search message if any > repeat for every user id in list of user ids 

Vs current 

Snipe appears > check if self or other > wait > snipe

**Comment by BlamoThePollo** [comment]
Oh wow that does seem a bit more complicated than how id imagine..
Thanks for the feedback though!
